kernel/kernel-auth-adapter/src/main/java/io/mosip/kernel/auth/defaultadapter/handler/AuthHandler.java

```diff
@@ -92,11 +103,33 @@
 
 	@Autowired
 	private ObjectMapper objectMapper;
-	
-	@Autowired
-	private RestTemplate restTemplate;
-	
+
+	private RestTemplate restTemplate = null;
+
 	private static final String DEFAULTADMIN_MOSIP_IO = "defaultadmin@mosip.io";

```

Comment:
What is this? I know u have not changed but why is it not in configuration with this as a default value? if this is not used please remove it.